### ISSUE TITLE
Import masterprivkey

### ISSUE THREAD

**Issue by navtrello** [NAVCoin/navcoin-core#113]

**Comment by aguycalled** [comment]
Use https://github.com/aguycalled/pycoin to derive addresses from a master private key.

```
git clone https://github.com/aguycalled/pycoin
cd pycoin
make
```

This will install the utility `ku`.

Example to derive 100 keys

```
$ ku -aW -s 0H/1H/1-100H -n NAV xprv9s21ZrDAJ8E9WQDjoiedjeuw9ioJFDEIOFJDEW9IjoikfjIEWODiKPMYyiuBGJHNd6
```

**Comment by antonkolesnyk** [comment]
Thank you! This is helpful. I just tried it, and one of the private keys was the private key or the address that I have used. This is great.
I have created a thread on Reddit earlier, with most of these questions. Some people are also interested to know the answers. I'll update it with the info you gave me.
Still, for the long term, I think that the best option would be to have a feature to create a wallet from a private master key, during the first launch. Right now, there are two options: to create a new wallet, or to import wallet.dat. I think if there was a third option to import a private master key - this would be ideal. And it would not be confusing, and unlikely error-prone.
The reason why this feature is still needed in my opinion is not only ease of use: right now, without wallet.dat backup, I only can create a new wallet and import a bunch of private keys from the old addresses. Now, if I transact from this new wallet, I might get funds into some addresses that were derived from the new master key, if I was not careful enough to specifically use an address that derives from the previous master key. And also it might give me the false sense of security - I might naively think that after I imported a bunch of private keys from the old addresses, and have written down the new master key, I will be able to restore all the funds, which won't be the case. So the best way for me to make sure I never loose any coins would be to write down master private key each time I reinstall wallet from scratch - and you see how quickly this might start requiring me to carry tens of private master keys instead of one.
